pallets/rmrk-core/src/lib.rs

```diff
@@ -219,6 +233,16 @@ pub mod pallet {
 	/// Lock for NFTs
 	pub type Lock<T: Config> = StorageMap<_, Twox64Concat, (CollectionId, NftId), bool, ValueQuery>;
 
+	#[pallet::storage]
+	pub type DummyStorage<T: Config> = StorageValue<
```

Comment:
Is it for generating the TypeInfo for `PropertyInfoOf<T>`? If so, please add some comments